packages/cli-platform-android/src/commands/runAndroid/index.ts

```diff
@@ -283,7 +287,6 @@ export default {
     {
       name: '--main-activity <string>',
       description: 'Name of the activity to start',
-      default: 'MainActivity',
```

Comment:
Maybe in future we'll deprecate this option, since right now it is automatically reading main activity. I have no idea how it works in e.g. brownfields setup. WDYT? cc. @thymikee